### PR TITLE
Added a couple handy debug vars for tracing

### PR DESCRIPTION
**Pull request by sglow**
Added a couple useful debug variables for tracing sensor readings and loop timing.

### CODE REVIEW

File: controller/lib/core/sensors.cpp [hunk 1]
```diff
@@ -137,8 +153,19 @@ SensorReadings Sensors::GetSensorReadings() {
...
+  dp_exhale = outflow_delta.cmH2O();
+  pressure = patient_pressure.cmH2O();
+
+  VolumetricFlow inflow = PressureDeltaToFlow(inflow_delta);
+  VolumetricFlow outflow = PressureDeltaToFlow(outflow_delta);
+  VolumetricFlow flow = inflow - outflow;
+
+  flow_inhale = inflow.liters_per_sec() * 1000.0f;
+  flow_exhale = outflow.liters_per_sec() * 1000.0f;
+  flow_delta = flow.liters_per_sec() * 1000.0f;
```
Comment (jlebar) [inline controller/lib/core/sensors.cpp:167]: Would you be willing to add a `TODO(jlebar): add ml_per_sec()`?

File: controller/lib/hal/hal_stm32.cpp [hunk 2]
```diff
@@ -341,12 +342,31 @@ void HalApi::startLoopTimer(const Duration &period, void (*callback)(void *),
...
+                   "%.2f");
+
 static void Timer15ISR() {
+  uint32_t start = TIMER15_BASE->counter;
   TIMER15_BASE->status = 0;
 
+  // Keep track of loop latency in uSec
+  // Also max latency since it was last zeroed
+  latency = static_cast<float>(start) * (1.0f / CPU_FREQ_MHZ);
+  max_latency = 0;
```
Comment (jlebar) [inline controller/lib/hal/hal_stm32.cpp:360]: Is this line a bug?

Comment (sglow) [reply controller/lib/hal/hal_stm32.cpp:360]: Oops, yes!  I know exactly how I did that, thanks for catching it.

File: controller/lib/core/sensors.cpp (hunk 1 above)
Comment (sglow) [reply controller/lib/core/sensors.cpp:167]: Good idea, I'll take care of that.

**Pull request merged**